src/aliceVision/sfmDataIO/viewIO.cpp

```diff
@@ -140,8 +140,8 @@ std::shared_ptr<camera::IntrinsicBase> getViewIntrinsic(
 
   camera::EINTRINSIC intrinsicType = defaultIntrinsicType;
 
-  double ppx = view.getWidth() / 2.0;
-  double ppy = view.getHeight() / 2.0;
+  double ppx = defaultPPx;
+  double ppy = defaultPPy;
```

Comment:
The default value was -1 (in the hpp) to say "no value", now -1 is a valid value.